markets/perps-market/contracts/storage/GlobalPerpsMarket.sol

```diff
@@ -73,13 +73,14 @@ library GlobalPerpsMarket {
         Data storage self
     ) internal view returns (uint128 rate, uint256 delegatedCollateralValue, uint256 lockedCredit) {
         uint256 withdrawableUsd = PerpsMarketFactory.totalWithdrawableUsd();
-        delegatedCollateralValue = withdrawableUsd - totalCollateralValue(self);
+        int256 delegatedCollateralValueAsInt = withdrawableUsd.toInt() -
+            totalCollateralValue(self).toInt();
         lockedCredit = minimumCredit(self);
-        if (delegatedCollateralValue == 0) {
+        if (delegatedCollateralValueAsInt <= 0) {
             return (0, 0, lockedCredit);
```

Comment:
If a market is undercollateralized, does it mean interest this rate factor here is zero? I think it might need to be set to1, because we kind of , need in this situation, folks to wind down their positions, so that the market becomes less volatile, for a "fix" to be proposed that re-establishes equilibrium is my intuitive take on it.

Reply:
confirmed with afif, set it to 1